pkg/chartutil/requirements.go

```diff
@@ -230,7 +231,7 @@ func getAliasDependency(charts []*chart.Chart, aliasChart *Dependency) *chart.Ch
 		if existingChart.Metadata.Name != aliasChart.Name {
 			continue
 		}
-		if existingChart.Metadata.Version != aliasChart.Version {
+		if !version.IsCompatibleRange(aliasChart.Version, existingChart.Metadata.Version) {
```

Comment:
would you mind writing up a unit test to cover this?

Reply:
got it,I have added it,please review it @bacongobbler